components/navigation/style.css

```diff
@@ -0,0 +1,48 @@
+a {
+    text-decoration: none;
+    color: whitesmoke;
+}
+
+ul li {
+    list-style-type: none;
+    font-size: 1.5rem;
+}
+
+.header {
+    color: whitesmoke;
+    background-color: blueviolet;
+}
+
+.header-flex {
+    padding:0 1rem;
+    margin-top: -0.5rem;
```

Comment:
Remove this margin as it breaks the alignment and things are not vertically aligned.

Reply:
okay update it.